app/src/main/java/net/squanchy/schedule/domain/view/Schedule.java

```diff
@@ -6,9 +6,14 @@
 
 @AutoValue
 public abstract class Schedule {
+
     public static Schedule create(List<SchedulePage> pages) {
         return new AutoValue_Schedule(pages);
     }
 
     public abstract List<SchedulePage> pages();
+
+    public boolean hasPages() {
```

Comment:
Again, I'd prefer a `isEmpty()` to not leak the concept of pages if not strictly needed